CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # go-ipfs changelog
 
+### 0.4.3-rc4 - 2016-09-06
+
+This release candidate fixes issues in Bitswap and the `ipfs add` command, and improves testing.
+We plan for this to be the last release candidate before the release of go-ipfs v0.4.3.
+
+With this release candidate, we're also moving go-ipfs to Go 1.7, which we expect to yield improvements in runtime performance, memory usage, build time and size of the release binaries.
+
+- Fix output of commands taking input from stdin. (@Kubuxu, [ipfs/go-ipfs#3134](https://github.com/ipfs/go-ipfs/pull/3134))
```

Comment:
Would this work? "Fix info notice of commands taking input from stdin."

To me *help* sounds like it's about the `--help` text.